src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/_params.py

```diff
@@ -120,6 +120,12 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
                    help='App properties of the route message.')
         c.argument('system_properties', options_list=['--system-properties', '--sp'],
                    help='System properties of the route message.')
+        c.argument('tags', options_list=['--tags'],
+                   help='Device twin tags.')
+        c.argument('desired', options_list=['--desired'],
+                   help='Device twin desired properties.')
+        c.argument('reported', options_list=['--reported'],
+                   help='Device twin reported properties.')
```

Comment:
You don't need any of the options lists here since the option is what the CLI would have derived anyways.